cmd/utils.go

```diff
@@ -108,6 +108,23 @@ func checkLatestVersion() {
 	}
 }
 
+func compareVersions(currentTag string, latestTag string) (bool, string) {
```

Comment:
Is there a particular reason for `(bool, string)` instead of `(string, bool)`?
I've mostly seen this sort of functions called like `response, ok := someFunc()`